src/app/data-model-provider/OperationTypes.h

```diff
@@ -73,6 +73,13 @@ enum class ReadFlags : uint32_t
     kFabricFiltered = 0x0001, // reading is performed fabric-filtered
 };
 
+enum class ListWriteOperation : uint8_t
+{
+    kListWriteBegin = 0,
+    kListWriteEnd,
+    kListWriteEndFinal
```

Comment:
I have no idea what the difference between those values is.  Needs documentation.

Reply:
if the difference if "writeWasSuccessful" variable, ListWriteSuccess/ListWriteFailure seems better. However then we need to describe what "failure" and "success" mean (unsure if the previous API really defined them either...)

Reply:
Yes, the "aWriteWasSuccessful" was the only difference since I wanted to account for that option, otherwise we could have only ListWriteBegin and ListWriteEnd. The current documentation in AAI is not really clear about the meaning of that variable.

Reply:
I mean, it's indicating whether the write actually succeeded or not, right?

Reply:
Sorry for the confusion, yes. Those should mean "failure" or "success", just got confused for the way it was worded in the original AAI documentation.